test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```

Comment:
again these files are not deleted?

DELETE all unused files

Reply:
done